src/web/index.ts

```diff
@@ -18,6 +18,16 @@ export type WebEngineAPI = BaseEngineAPI & Pick<
 	getState(): GameState // For weird backwards-compatibility reasons, not part of API
 }
 
+const SPRITE_CACHE_SIZE = 64;
```

Comment:
This value was chosen after determining that 16 and 32 didn't have a noticable impact.